builds/e2e/benchmark.yaml

```diff
@@ -0,0 +1,232 @@
+trigger: none
```

Comment:
I think this file should not be in e2e folder.  If you foresee more yamls for benchmark, we should create a new folder named "benchmark". if not, maybe move it to misc folder.